lib/std/crypto/bcrypt.zig

```diff
@@ -178,6 +179,88 @@ pub fn bcrypt(
     return ct[0..dk_length].*;
 }
 
+const pbkdf_prf = struct {
```

Comment:
> [ One modification from official pbkdf2. Instead of outputting key material linearly, we mix it.
pbkdf2 has a known weakness where if one uses it to generate (i.e.) 512 bits of key material for use as two 256 bit keys, an attacker can merely run once through the outer loop below, but the user always runs it twice. Shuffling output bytes requires computing the entirety of the key material to assemble any subkey. This is something a wise caller could do; we just do it for you.](https://github.com/grnet/python-bcrypt/blob/master/bcrypt/bcrypt_pbkdf.c#L146)

Something I stumbled on.